easybuild/easyconfigs/m/mcpl/mcpl-2.2.0-foss-2023a.eb

```diff
@@ -0,0 +1,39 @@
+# Author: Erik B Knudsen
+# United Neux
+
+easyblock = 'CMakeMake'
+
+name = 'mcpl'
+version = '2.2.0'
+
+homepage = 'https://mctools.github.io/mcpl'
+description = "MCPL (Monte Carlo Particle Lists) is library and a file format for storing " \
+    "Monte Carlo particles to be used with a range of particle transport packages, " \
+    "such as MCNP, OpenMC, PHITS, McStas, McXtrace, as well as supporting tools such as KDSource"
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+github_account = 'mctools'
+source_urls = ['https://github.com/mctools/mcpl/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['0cd926621d7d45d2bb39fab64732448b67f622cda9434a105de4c9d942a3db5c']
+
+builddependencies = [
+    ('CMake', '3.26.3'),
```

Comment:
```suggestion
    ('CMake', '3.31.8', '', SYSTEM),  # Needs CMake > 3.28 for testing
```

Reply:
`SYSTEM` means we use the system compilers instead of a specific toolchain, but we still build the required version of CMake in this case.  The assumption is that CMake can be built with the system compilers.

The alternative would be to add also a newer version of CMake to `foss/2023a` + an exception in the CI tests.
Will check if a similar problem has already been solved with a preferred way

Reply:
Ah OK - so if I understand correctly (please correct me if I am wrong) in this case cmake 3.31.8 (from source) will be built using whatever is available on the system, and used instead of 3.26 from foss 2023a. This works since it's a build req.